application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCommand.java

```diff
@@ -0,0 +1,145 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.unions.MessageChannelUnion;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+
+import java.util.Objects;
+
+public final class BookmarksCommand extends SlashCommandAdapter {
+
+    private static final Logger logger = LoggerFactory.getLogger(BookmarksCommand.class);
+
+    public static final String COMMAND_NAME = "bookmarks";
+    public static final String SUBCOMMAND_ADD = "add";
+    public static final String SUBCOMMAND_LIST = "list";
+    public static final String SUBCOMMAND_REMOVE = "remove";
+    public static final String ADD_OPTION_NOTE = "note";
+
+    private static final MessageEmbed NOT_A_HELP_THREAD_EMBED =
+            BookmarksSystem.createFailureEmbed("This command can only be run in help threads!");
+
+    private static final MessageEmbed ALREADY_BOOKMARKED_EMBED =
+            BookmarksSystem.createFailureEmbed("You already bookmarked this channel!");
+
+    private static final MessageEmbed BOOKMARK_ADDED_EMBED =
+            BookmarksSystem.createSuccessEmbed("Bookmark added!");
+
+    private static final MessageEmbed BOOKMARK_LIMIT_USER_EMBED =
+            BookmarksSystem.createFailureEmbed(
+                    "You have exceeded your bookmarks limit! Please delete your old bookmarks.");
+
+    private static final MessageEmbed BOOKMARK_LIMIT_TOTAL_EMBED =
+            BookmarksSystem.createWarningEmbed("""
+                    You can't add a bookmark right now because the total amount
+                    of bookmarks has exceeded its limit! Please wait a bit.
+                    """);
+
+    private final BookmarksSystem bookmarksSystem;
+
+    /**
+     * Creates a new instance and registers every sub command.
+     *
+     * @param bookmarksSystem The bookmarks system to use
+     */
+    public BookmarksCommand(BookmarksSystem bookmarksSystem) {
+        super(COMMAND_NAME, "Bookmark help threads", CommandVisibility.GLOBAL);
```

Comment:
maybe add some motivation to the description. `Bookmark help threads, so that you can easily look them up again`